server/util.js

```diff
@@ -62,10 +62,106 @@ const coordinatesDistanceCalc = (lat1, lng1, lat2, lng2, unit = "K") => {
     return dist;
 };
 
+// src: https://stackoverflow.com/questions/36506668/google-static-map-draw-a-circle
+const drawCirclePath = (lat, lng, radius, detail = 8) => {
+    let R = 6371;
+
+    let pi = Math.PI;
+
+    lat = (lat * pi) / 180;
+    lng = (lng * pi) / 180;
+    let d = radius / R;
+
+    let points = [];
+    let i = 0;
+
+    for (i = 0; i <= 360; i += detail) {
+        let brng = (i * pi) / 180;
+
+        let plat = Math.asin(
+            Math.sin(lat) * Math.cos(d) + Math.cos(lat) * Math.sin(d) * Math.cos(brng)
+        );
+        let plng =
+            ((lng +
+                Math.atan2(
+                    Math.sin(brng) * Math.sin(d) * Math.cos(lat),
+                    Math.cos(d) - Math.sin(lat) * Math.sin(plat)
+                )) *
+                180) /
+            pi;
+        plat = (plat * 180) / pi;
+
+        let currentPoints = [plat, plng];
+        points.push(currentPoints);
+    }
+
+    return createEncodings(points);
+}
+
+function createEncodings(coords) {
```

Comment:
Let's clean this file up to be consistent with our code style: 1) how functions are declared, 2) let/const/var

Reply:
Agree, fixed it